Link Review Getting Started

eliasmbd opened the pull request:
- Reviewed all links in Getting started section
- harmonized format of links to match ../, did not change format in feature files

eliasmbd committed:
Link Review Getting Started 

- Reviewed all links in Getting started section
- harmonized format of links to match ../, did not change format in feature files

eliasmbd committed:
Link check why_use_GTFS.md

eliasmbd committed:
Link check create.md

eliasmbd committed:
link check validate.md

eliasmbd committed:
Link Check publish.md

eliasmbd committed:
Link check overview.md

eliasmbd committed:
Link check base.md

eliasmbd committed:
Link check accessibility.md

eliasmbd committed:
Link check fares.md

eliasmbd committed:
Link check pathways.md

Sergiodero committed:
Fix two links in features overview

Sergiodero committed:
Fixed prerequisites format for continuous stops

Sergiodero committed:
Update Github reference to direct to google/transit

Sergiodero reviewed (approved):
Thanks @eliasmbd! I've finished my review and added some commits to complement your changes, It looks good to me so I'm approving. 

One last thing that I wanted to check with you is that I updated a reference to GitHub in the _Why GTFS?_ page to direct users to google/transit instead of GitHub main page (see [latest commit](https://github.com/MobilityData/elrond.gtfs.org/pull/76/commits/c0e2a1444930d5defe7b3dd5c9fc87e2f10704c1)). I think this is more context appropriate and harmonizes this reference to the slack one directing users to sign-up for MobilityData slack channels. What do you think?

eliasmbd commented:
> Thanks @eliasmbd! I've finished my review and added some commits to complement your changes, It looks good to me so I'm approving.
> 
> One last thing that I wanted to check with you is that I updated a reference to GitHub in the _Why GTFS?_ page to direct users to google/transit instead of GitHub main page (see [latest commit](https://github.com/MobilityData/elrond.gtfs.org/pull/76/commits/c0e2a1444930d5defe7b3dd5c9fc87e2f10704c1)). I think this is more context appropriate and harmonizes this reference to the slack one directing users to sign-up for MobilityData slack channels. What do you think?

Makes sense to me. I think i missed that one

Sergiodero commented:
Perfect, I'm merging now then, thanks again!
